cmd/uplink/cmd/setup.go

```diff
@@ -124,12 +142,13 @@ Please enter numeric choice or enter satellite address manually [1]: `)
 		if apiKey == "" {
 			return errs.New("API key cannot be empty")
 		}
+		override["api-key"] = apiKey
 
 		_, err = fmt.Print("Enter your encryption passphrase: ")
 		if err != nil {
 			return err
 		}
-		encKey, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+		encryptionKey, err = terminal.ReadPassword(int(os.Stdin.Fd()))
```

Comment:
nit: storj.NewKey takes a passphrase, so maybe this variable should be passphrase? fine to leave as-is